hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/LegacyHoodieParquetFileFormat.scala

```diff
@@ -23,12 +23,12 @@ import org.apache.hudi.{DataSourceReadOptions, HoodieSparkUtils, SparkAdapterSup
 import org.apache.spark.sql.SparkSession
 import org.apache.spark.sql.catalyst.InternalRow
 import org.apache.spark.sql.execution.datasources.PartitionedFile
-import org.apache.spark.sql.execution.datasources.parquet.HoodieParquetFileFormat.FILE_FORMAT_ID
+import org.apache.spark.sql.execution.datasources.parquet.LegacyHoodieParquetFileFormat.FILE_FORMAT_ID
 import org.apache.spark.sql.sources.Filter
 import org.apache.spark.sql.types.{AtomicType, StructType}
 
 
-class HoodieParquetFileFormat extends ParquetFileFormat with SparkAdapterSupport {
+class LegacyHoodieParquetFileFormat extends ParquetFileFormat with SparkAdapterSupport {
```

Comment:
Add docs here to link to the new file format implementation so that any changes to this format implementation should also reflect in the new file format class?

Reply:
I think changes to the relation and especially rdd classes are more likely to require changes in the new file format than changes to LegacyHoodieParquetFileFormat.

Reply:
Yeah, my point is developers changing this class should be aware of the new file format and not miss changing the new file format causing inconsistency, if necessary.